wordpress-mobile/WordPress-Android
# Updater picker empty list message to match media type

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/3167

To test:
0. Choose a site with no media in the WordPress media library 
1. Open up the editor and add one of each blocks to a post: Image, Video, Audio and Media
2. For each block try adding a file from the "WordPress  Media Library"
3. Verify that the empty media message matches the block type i.e. Image block -> "You don't have any images", etc 
4. Open the post on a device with no media ( such as an emulator )
5. For each block try adding a file via "Choose from device"
6. Verify that the empty media message matches the block type.

## Regression Notes
1. Potential unintended areas of impact
This touches how the media picker collects files and could impact non empty file systems


3. What I did to test those areas of impact (or what existing automated tests I relied on)
I tried the picker on sites with and with out media 

8. What automated tests I added (or what prevented me from doing so)
None were added. 

PR submission checklist:
